AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -98,6 +98,9 @@ @interface ASCollectionView () <ASRangeControllerDelegate, ASDataControllerSourc
   ASDataController *_dataController;
   ASRangeController *_rangeController;
   ASFlowLayoutController *_layoutController;
+
+  BOOL _batchUpdateFlag;
```

Comment:
Hmm, I will rename it to be `_performingBatchUpdates.`
The `_batchUpdateBlocks.count > 0` is not enough for the checking considering it is changed after layout in background thread.